src/components/molecules/content/blog-elements/BlogFooter.svelte

```
@@ -0,0 +1,16 @@
+<footer class="border-t border-gray-200 px-10 py-12">
+	<div class="mx-auto max-w-7xl">
+		<div class="flex flex-col items-center justify-between md:flex-row">
+			<div class="flex items-center gap-3">
+				<img src="/apple-touch-icon.png" alt="Poesty.ai" class="h-8 w-8" />
+				<span class="text-xl font-semibold text-gray-800"> Poesty.ai </span>
+			</div>
+
+			<div class="flex items-center gap-6 text-sm text-gray-600">
+				<a href="/privacy" class="transition-colors hover:text-blue-600">Privacy</a>
```

ESLint

[error] Found a link with a url that isn't resolved.

(svelte/no-navigation-without-resolve)